### PR TITLE
Pass the name of HTTP method to mocking callback

### PR DESCRIPTION
**Pull request by maxkoryukov**

### CODE REVIEW

**Comment by tbranyen** [comment]
No unit test for this? How is it used?

**Comment by maxkoryukov** [comment]
@tbranyen , you are right, no need to include this into upstream. I have a very convenient mocks function for this;)
Just publish **0.5.3** on npm, please, we are wating...

File: package.json [hunk 1]
```diff
@@ -22,6 +22,12 @@
     "test"
   ],
   "author": "Anton <anton@shuvalov.info> (http://shuvalov.info/)",
+  "contributors": [
+    {
+      "name": "Koryukov Maksim",
+      "email": "maxkoryukov@gmail.com"
+    }
+  ],
```
Comment (A) [inline package.json]: Why https://github.com/A/superagent-mocker/graphs/contributors isn't enough for you?

Comment (A) [inline package.json]: -1 from me

**Comment by A** [comment]
Please update the docs for update. Seems good to me.

**Comment by A** [comment]
Sorry guys, I just missed status of the PR)

**Comment by maxkoryukov** [comment]
I have revoked this PR - this amendments are useful only for my old approach, not for everyone..

@A, plz, better look at #47 - it is already merged, but the latest NPM version is **0.5.2** - without fixes

**Pull request closed without merging**